core/src/main/scala/scalaz/package.scala

```diff
@@ -114,6 +114,7 @@ package object scalaz {
   type |>=|[G[_], F[_]] = MonadPartialOrder[G, F]
 
   type ReaderT[F[_], E, A] = Kleisli[F, E, A]
+  val ReaderT = Kleisli
```

Comment:
Hmm we have this https://github.com/scalaz/scalaz/blob/b55570be26de7051d9ed8904d7effeae34ed52cf/core/src/main/scala/scalaz/package.scala#L124-126

or did you have something else in mind?